### ISSUE TITLE
Storybook MaskedInput E-Mail example does not allow .co.uk or dashes in domain name

### ISSUE THREAD

**Issue by Zahlenspiel** [grommet/grommet#6225]
### Expected Behavior
https://storybook.grommet.io/?path=/story/input-maskedinput-email-with-icon--email-masked-input
MaskedInput should allow endings like .co.uk and "-" in domain like "example@example-test.co.uk

### Actual Behavior
Gets masked.

### URL, screen shot, or Codepen exhibiting the issue
https://storybook.grommet.io/?path=/story/input-maskedinput-email-with-icon--email-masked-input
[Storybook Screenshot](https://prnt.sc/PmPvaLklTOAG)

### Steps to Reproduce

Use https://storybook.grommet.io/?path=/story/input-maskedinput-email-with-icon--email-masked-input

### Your Environment
Storybook

**Comment by nomadus** [comment]
Yes, the mail mask has restrictions, I'd suggest making it less restrictive, e.g. use

```
let emailMask = [
			{
				regexp: /^.+$/,
				placeholder: "example",
			},
			{fixed: "@"},
			{
				regexp: /^.+$/,
				placeholder: "my",
			},
			{fixed: "."},
			{
				regexp: /^.+$/,
				placeholder: "com",
			},
		];
```
